.travis.yml

```diff
@@ -0,0 +1,32 @@
+notifications:
+  email: false
+  slack:
+    # Slack integration.
+    # Generated via `travis encrypt -r ClusterHQ/dvol <slack configuration>'
+    # See Travis in the Slack App directory for details.
+    secure: "IuMdO5YgVlYdq12I8PbrnjgQo2mpW53LoABUHoHFnaczpXmOD0cbgSX4YRRqGPTkBj3iMqOYtGbOtcyIRwLB9SVbr2xeDpvRYC8auX/jGMbQmaQMhG4kZOpVA9FKt+XcJFj6lsihT50tx2uJ4zn85KjOy1ZJQjgpka+vsKJbvGAg65AvCovlB/l1fyd8eaRZxDXyvapD7I39R7ij22r/QfnJ0t2JT0PKS0SR+gPemnjMhGkW0n3dyoOOkhqVAEyevt6f+s8NNnSioCuJy6X9R6eIXOeJYXHMvigxpZ7nEnrWik3IgQ9+rqJjjqzb9paxU+1+aQP0gb+r1M+hfbGyWQp1k6KCNb1IljCtjNccWxlajkooLJS1haRKiyV96U+DiDybpVIbEdtNMBQrRN+6lCbU8OduHxluH5WnAdZEUGz6pQLt7fxCazKzib/vxnUlSgvRnKzweYd21zj3c4w6aPLdCaI2eQM036Zwt/9ltWbk5I8sjAWfCgvWqxD+MaOT2vAog8T7B8WoyRjhbChtMNNHtw64a9W402WADrQO1jTU4tGsKwDbjXb4j2GtvhKahrGGigFYBfPLgOi8mzzwaJh7uBo/S8evAG/K/abc2hhhLBGWlSiA252hbNcgjGk8QuwKFhaA1DenAXZTf8GcwYC+ZlmKOczIujamQKfAvjM="
+
+language: python
+python: 2.7
+sudo: false
```

Comment:
This will run in a docker container, not an OpenVZ container.
hopefully that's what you're looking for